packages/forms/src/components/Fields/Fields.components.tsx

```diff
@@ -83,6 +83,19 @@ export const Fields: React.FC<Props> = React.memo(props => {
                                 switch (componentProps.type) {
                                     case 'text':
                                     case 'email':
+                                    case 'url':
+                                    case 'tel':
+                                    case 'time':
+                                    case 'month':
+                                    case 'week':
+                                    case 'range':
+                                    case 'reset':
```

Comment:
Actually for now `TextField` component will not support `reset`, `image`, `hidden`, `button` & `submit`.

Reply:
Removed